src/components/qrModal/qrModalView.tsx

```diff
@@ -192,13 +203,19 @@ export const QRModal = ({}: QRModalProps) => {
               id: 'qr.approve',
             }),
             onPress: () => {
-              handleHiveUriOperation(currentAccount, pinCode, uri)
+              handleHiveUriOperation(currentAccount, pinCode, tx)
```

Comment:
now that we have started to convert uri to operations array, can we make use of existing op array broadcaster instead of introducing `handleHiveUriOperation`... thoughts...?

Reply:
I think handling array operations would add complexity, also hivesigner do not generate uris for array ops